package.json

```diff
@@ -30,7 +30,7 @@
     "clean": "rm -rf dist",
     "lint": "eslint --cache --quiet --fix src",
     "lint-ci": "eslint --quiet src",
-    "semantic-release": "ht2-release-public-circleci-app",
+    "semantic-release": "ht2-release-public-circleci-lib",
```

Comment:
What was the reason this was changed? It is actually an app rather than a library.

Reply:
THat's what the call later is to discuss